src/amberscript/parser_shader_test.cc

```diff
@@ -235,6 +235,26 @@ END)";
   EXPECT_EQ("2: extra parameters after SHADER command: INVALID", r.Error());
 }
 
```

Comment:
Add a test for duplicate virtual file name

Reply:
Ack

Reply:
Done. Also added one for blank paths.